src/test/java/uk/gov/companieshouse/model/validator/AppealReasonValidatorTest.java

```diff
@@ -0,0 +1,48 @@
+package uk.gov.companieshouse.model.validator;
+
+import static org.junit.jupiter.api.Assertions.assertDoesNotThrow;
+import static org.junit.jupiter.api.Assertions.assertThrows;
+import static uk.gov.companieshouse.util.TestUtil.createIllnessReason;
+import static uk.gov.companieshouse.util.TestUtil.createOtherReason;
+
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.api.extension.ExtendWith;
+import org.mockito.InjectMocks;
+import org.mockito.Mock;
+
+import org.springframework.test.context.junit.jupiter.SpringExtension;
+import uk.gov.companieshouse.exception.AppealReasonException;
+import uk.gov.companieshouse.model.Reason;
+
+@ExtendWith(SpringExtension.class)
+class AppealReasonValidatorTest {
+
+    @Mock
+    private Reason mockReason = new Reason();
+
+    @InjectMocks
+    private AppealReasonValidator appealReasonValidator;
+
+    @Test
```

Comment:
FYI unit tests in JUnit 5 can have readable descriptions.

```
@DisplayName("Validation failure for no Appeal reasons")
@Test
void shouldFailValidationForNoReasons()

@DisplayName("Validation failure for more than one Appeal reason")
@Test
void shouldFailValidationForMultipleReasons()

@DisplayName("Validation success for other Appeal reason only")
@Test
void shouldPassValidationForOtherReason()

@DisplayName("Validation success for illness Appeal reason only")
@Test
void shouldPassValidationForIllnessReason()
```

Reply:
Thanks for the tips :)